kt-extensions/src/main/java/com/mindera/skeletoid/kt/extensions/utils/CurrencyFormatter.kt

```diff
@@ -12,8 +12,9 @@ import java.util.Locale
  */
 fun String?.parseCurrency(locale: Locale = Locale.UK, convertCents: Boolean = false): Float? {
     return this?.let { amount ->
-        val value = NumberFormat.getInstance(locale).parse(amount)?.toFloat() ?: throw NumberFormatException()
-        return if (convertCents) value / 100f else value
+        val value = runCatching { NumberFormat.getInstance(locale).parse(amount)?.toFloat() }.getOrNull()
```

Comment:
what's the advantage of having it in runCatching and then throwing exception?

Reply:
This way we can control what exception do we throw. With the old code I don't even know how to mock a case where we get to throw `NumberFormatException` because if the parsing fails we get a parsing exception